File: program (112).java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    Multiply
//  Description :       Return the multiplication of all digits. 
//  Input :             int 
//  Output :            int
//  Author :            Aman Shaikh    
//  Date:               14/12/2024 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Digit
{
    public int Multiply(int iNo)
    {
        int i = 0, iCnt = 1;
        while(iNo != 0)
        {
            i = iNo % 10;
            if(i != 0)
            {
                iCnt = iCnt * i;
            }
            iNo = iNo / 10;
        }
        return iCnt;
        
    }
}

class Assignment38D
{
    public static void main(String Arr[])
    {
        int iValue = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter number : ");
        iValue = sobj.nextInt();  

        Digit aobj = new Digit();
        int iResult = aobj.Multiply(iValue); 

        System.out.println("Multiplication is : "+iResult);
    }
}

File: program (112).c
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Function Name  :    Display
//  Description :       Accept 'N' number from user and convert smaller case into upper case and vice-versa. 
//  Input :             char 
//  Output :            char
//  Author :            Aman Shaikh    
//  Date:               19/11/2024
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Display(char cVal)
{

        if(cVal >= 65 && cVal <= 90)
        {
            printf("Your Output is : %c\n",cVal+32);
        } 
        else if(cVal >= 97 && cVal <= 122)
        {
            printf("Your Output is : %c\n",cVal-32);
        }
        else
        {
            printf("Your Output is : %c\n",cVal);
        }
}

int main()
{
    char cValue = '\0';

    printf("Enter the character : ");
    scanf("%c",&cValue);   

    Display(cValue);

    return 0;
}